_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
> If we require Jekyll ~> 3.3, I think we can make this a non-breaking change. 

By breaking change I was implying that this would cause trouble unless the user manually intervenes and changes their site's source code. Sorry, if the mainstream definition implies something else.
`require`ing ~>3.3  will not solve things as of now..